ionos_sbx/marketplace/access-node/values.yaml

```diff
@@ -24,45 +28,45 @@ access-node:
       ## configuration of ngsi-ld entities to listen for
       ngsiSubscription:
         # -- local address of the blockchain-connectors notification endpoint for ngsi-ld events
-        notificationEndpoint: http://desmos:8080/notifications/broker
+        notificationEndpoint: /api/v1/notifications/broker
         # -- a list of entity-types the connector is interested in
         entityTypes: catalog,product-offering,category,individual,organization,product,service-specification,product-offering-price,resource-specification,product-specification
       ## configuration to connect to the dlt-adapter
       dltAdapter:
         # -- provider of the dlt-adapter component
         provider: digitelts
         # -- (local) address of the dlt-adapter
-        internalDomain: http://dlt-adapter:80
+        internalDomain: http://dlt-adapter:8080
         # -- (external) address of the dlt-adapter
-        externalDomain: http://dlt-adapter:80
+        externalDomain: http://dlt-adapter:8080
         ## configuration of subscriptions at the blockchain
       txSubscription:
         # -- local address of the blockchain-connectors notification endpoint for dlt events
-        notificationEndpoint: http://desmos:8080/notifications/dlt
+        notificationEndpoint: /api/v1/notifications/dlt
         # -- a list of entity-types the connector is interested in
         entityTypes: catalog,product-offering,category,individual,organization,product,service-specification,product-offering-price,resource-specification,product-specification
     ## configuration of the database to be used by the blockchain-connector
     db:
       # -- host of the db
-      host: pg-75qkrjg6i4d7tlg2.postgresql.de-txl.ionos.com
+      host: postgresql-connector
```

Comment:
Why do you refer to 'postgresql-connector' instead of 'pg-75qkrjg6i4d7tlg2.postgresql.de-txl.ionos.com' . Did you define somewhere a link between those 2 hosts?